Cargo.toml

```diff
@@ -35,9 +36,11 @@ time = { version = "0.3", default-features = false }
 sha1 = { version = "0.10", default-features = false }
 num-derive = "0.4"
 num-traits = { version = "0.2", default-features = false }
-picky = { version = "7.0.0-rc.11", default-features = false }
-picky-asn1-der = "0.5"
-picky-asn1-x509 = "0.14"
+picky = { git = "https://github.com/Devolutions/picky-rs.git", branch = "master"}
+picky-asn1 = { git = "https://github.com/Devolutions/picky-rs.git", branch = "master"}
+picky-asn1-der = { git = "https://github.com/Devolutions/picky-rs.git", branch = "master"}
+picky-asn1-x509 = { git = "https://github.com/Devolutions/picky-rs.git", branch = "master"}
+picky-krb = { git = "https://github.com/Devolutions/picky-rs.git", branch = "master"}
```

Comment:
note: I’ll publish the crates before we merge.

Reply:
I’m sorry for the delay. I’m automating the publishing process, this should be ready shortly.